fdbclient/admin_client.go

```diff
@@ -345,6 +345,34 @@ func (client *cliAdminClient) ConfigureDatabase(configuration fdbv1beta2.Databas
 	return err
 }
 
+// GetMaintenanceZone gets current maintenance zone, if any. Returns empty string if maintenance mode is off
+func (client *cliAdminClient) GetMaintenanceZone() (string, error) {
+	status, err := client.GetStatus()
```

Comment:
Do you know if there's a special key that could be used to read this information? Fetching the whole status only for one key seems sub-optimal.

Reply:
I think there is one.

Reply:
I will add this as a TODO for future improvements. Since it is going to be off by default, we can address it before it is enabled.